README.md

```diff
@@ -2,7 +2,7 @@
 
 The purpose of this gem is to allow VFS engineers to quickly begin developing on the platform. It does this by providing a command line interface that allows the use of simple commands and parameters to do everything from setting up a development environment to building out a directory structure and creating necessary files for separating code into its own module.
 
-*The following assumes you have Ruby 2.6.6 or higher installed.*
+*The following assumes you have Ruby 2.6.6 or higher and Rails ~> 6.0.2 installed*
```

Comment:
I don't think Rails should be a requirement for the gem. It should be using the project's `bin/rails` instead of system  rails.

Reply:
removed